reduce.js

```diff
@@ -8,3 +8,6 @@ var numbers = [1, 56, 2, 4, 1, 99, 3, 5];
 // via multiplication
 
 
+const reducer = (accumulator, currentValue) => accumulator + currentValue;
+
+console.log(numbers.reduce(reducer));
```

Comment:
nice work with `reduce()` A couple of notes:

* small detail, but the instructions say to get the **product**, not the sum, of the numbers 😄 
* a question to make sure you have an answer to: what is the default value of the `accumulator` if you don't specify it?